backend/src/v4/routes/model.js

```diff
@@ -1714,7 +1714,7 @@ router.patch("/:model/upload/ms-chunking/:corID", middlewares.hasUploadAccessToM
  * ------WebKitFormBoundarySos0xligf1T8Sy8I-- *
  *
  */
-router.post("/:model/upload", middlewares.hasUploadAccessToModel, uploadModel);
+router.post("/:model/upload", middlewares.chechPermsAndFormatV5NewRevisionsData, validateNewRevisionData, uploadModel);
```

Comment:
Let's keep them separate 
```suggestion
router.post("/:model/upload", middlewares.hasUploadAccessToModel, middlewares.formatV5NewRevisionsData, validateNewRevisionData, uploadModel);
```